### DISCUSSION TITLE
Choice selected by mouse that has not moved

### DISCUSSION THREAD

**Discussion by hakubo** [johnlindquist/kit#261] [Errors]
Often I choose a wrong choice in the dropdown because my mouse pointer just happens to be over an option and selects something. This happens when mouse does not move.

Hope that this video will showcase that:
https://user-images.githubusercontent.com/1018759/117931332-59ec6000-b2ff-11eb-8b9b-16ab145ce343.mov

**Comment by johnlindquist** [comment]
Yeah, I experience the same thing. It's super annoying. I've tried to fix this four or five different ways.

I think completely disabling mouse events on the window until a resize event fires will work. I have a couple other ideas. I'll figure out something 😬

**Reply by hakubo** [reply]
does "mousemove" fire in this case? Maybe 'enable' mouse only if after this fires?

**Reply by johnlindquist** [reply]
It does, yes. I've tried all the mouse events.

**Reply by johnlindquist** [reply]
I think I got it sorted by waiting for a "mouse enter" on the scroll area which then enables the mouse handlers on the "Choice Buttons". Pushing out a build tomorrow.
